dev/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM ubuntu:20.04
+SHELL ["/bin/bash", "-c"]
+
+# tools
+RUN su - \
```

Comment:
I never need su or sudo on an ubuntu image, why do you need it?

Reply:
removed.